doc/user-guide/src/installation/other.md

```diff
@@ -7,7 +7,7 @@ by platform:
   `x86_64-pc-windows-msvc` target][setup]. In general, this is the build of
   `rustup` one should install on Windows. This will require the Visual C++
   Build Tools 2019 or equivalent (Visual Studio 2019, etc.) to already be
-  installed. If you would prefer to install GNU toolchains or the i686
+  installed. If you would prefer to install GNU toolchains or the i686 or aarch64
```

Comment:
I think this is not the intended way. If Windows on ARM64 lacks proper host triple detection, we should fix it instead of documenting the wrong behavior.

Reply:
No, it detects the host triple correctly. This was useful for x64 emulated Rustup, but is likely not needed anymore if the right artefact is published.

Reply:
I’ll rephrase this section; right now it recommends to use x64 Rustup in all cases, but I think since Ring workaround is not needed anymore there’s no reason not to recommend the aarch64 Rustup to Windows Arm users.

Reply:
> No, it detects the host triple correctly. This was useful for x64 emulated Rustup, but is likely not needed anymore if the right artefact is published.

@Alovchin91 Let's track this in https://github.com/rust-lang/rustup/issues/3514.

Reply:
Rephrased. I'm actually not sure if #3514 is relevant here, since I don't understand the scenario where one would choose to use emulated Rustup with a native toolchain 🤔 It might be useful though to let Rustup upgrade itself from an emulated to a native version.

Reply:
@Alovchin91 Imagine one uses another app that installs Rustup. I don't know for Windows, but on macOS the guarantee is that once x64 is in the call chain, the rest of it is under emulation, so this will clearly cause problems. However this is out of the scope of this PR anyway.